config/eleventy/filters.js

```diff
@@ -41,4 +41,12 @@ module.exports = {
     const { locale } = this.ctx;
     return array.find(item => item.locale === locale);
   },
+
+  inferAltSlug(languageAlts, altLocale) {
+    const baseString = `/${altLocale}/`;
+    if (!languageAlts) return baseString;
+    const dict = JSON.parse(languageAlts);
+    if (!dict[altLocale]) return baseString;
+    return `${baseString}${dict[altLocale]}/`;
+  },
```

Comment:
Guess it's taste or style, but in my opinion that func is not so easy to read. How about decompressing it a bit?

```js
inferAltSlug(languages, locale) {
    const baseString = `/${locale}/`;
    if (!languages) {
      return baseString
    }
    
    const dict = JSON.parse(languages);
    const { locale: slug } = JSON.parse(languages);
    if (!slug) {
      return baseString
    }
    
    return `${baseString}${slug}/`;
}
```

Reply:
I agree - I'll have a look when I'm back on the job